packages/fxa-payments-server/src/components/PaymentForm/index.tsx

```diff
@@ -244,7 +248,10 @@ export const PaymentForm = ({
 
       <div className="legal-blurb">
         <p>Mozilla uses Stripe for secure payment processing.</p>
-        <p>View the <a href="https://stripe.com/privacy">Stripe privacy policy</a>.</p>
+        <p>
+          View the{' '}
```

Comment:
Is `{' '}` because you'd like more space before the link?

Reply:
i kinda think prettier added that...it's not necessary